### ISSUE TITLE
Ability to set the default morphological analysis for a form in user preferences

### ISSUE THREAD

**Issue by balmas** [latin-language-toolkit/arethusa#599]
Requested by Vanessa:

It would be nice if one could personally set the default form of every morphology or at least the order of the choices.  As far as I can tell, they come up basically alphabetically.  This results in odd situations where, for example, the extremely obscure dual forms show up before the normal singular forms, and the default genitive plural of nouns is the least common choice (feminine).  Also, for example, the dative singular of polis (city), which is a fairly common word, is the last of 8 or 9 items, most of which are pretty obscure.  The same situation obtains for the common preposition παρά and the common adverb “later” which is buried long after forms for the word for uterus.  Again, I not only have the extra click to choose what is by far the most common usage, but I also have to scroll off the bottom of my screen to even find that common usage.

**Comment by LFDM** [comment]
We're really just showing the order we get from the morphology service - ordering frequency would be the killer feature here...

**Comment by balmas** [comment]
yes that would be lovely, but maybe we could just remember a user's choices for now?
